village/src/components/SmurfForm.js

```diff
@@ -13,7 +14,17 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
-
+    const smurf = {
```

Comment:
Excellent job building this object separately first, then passing it to the post.